src/cljs-lib/src/calva/repl/webview/core.cljs

```diff
@@ -0,0 +1,229 @@
+                           "REPL Output"
+                           #js {:preserveFocus true
+                                :viewColumn (.. ^js vscode -ViewColumn -Beside)}
+                           #js {:enableScripts true
+                                ;; If performance or memory consumption becomes a problem, we can use the setState
+                                ;; and getState to manually retain the context of the webview when it's hidden.
+                                ;; See https://code.visualstudio.com/api/extension-guides/webview#persistence
+                                :retainContextWhenHidden true
+                                :enableFindWidget true}))]
+    (.. ^js webview-panel (onDidDispose (fn [] (dispose-repl-output-webview-panel repl-output-webview-panel))))
+    (set-webview-html! context {:webview-panel webview-panel})
+    (add-subscriptions! context {:webview-panel webview-panel})
+    webview-panel))
+
+(defn ^:export show-repl-output-webview-panel
+  [preserve-focus?]
+  (let [context {:env/is-debug (:is-debug util/env)
+                 :vscode/vscode @util/vscode
+                 :vscode/context @util/vscode-context}
+        ^js webview-panel (or @repl-output-webview-panel
+                              (reset! repl-output-webview-panel (create-repl-output-webview-panel context)))
+        active-code-theme-kind (.. ^js @util/vscode -window -activeColorTheme -kind)]
+    (.. webview-panel (reveal nil preserve-focus?))
+    (set-code-theme! context {:color-theme-kind active-code-theme-kind
+                              :webview-panel webview-panel})))
+
+(def output-category->command-name
+  {"otherOut" "show-stdout"
+   "evalOut" "show-stdout"
+   "evalResults" "show-result"
+   "evalErr" "show-stdout"
+   "otherErr" "show-stdout"
+   "clojure" "show-result"})
+
+(defn ^:export append
+  [^js options message]
+  (let [output-category (.-outputCategory options)
+        command-name (get output-category->command-name output-category)]
+    (if command-name
+      (post-message-to-webview @repl-output-webview-panel {:command/name command-name
+                                                           :content message})
+      (util/log-to-console
+       :error
+       (str "Cannot append content to output webview. No outputCategory matches \"" output-category "\"")))))
+
+(def stacktrace-classes-to-ignore
+  #{"clojure.lang.RestFn"
+    "clojure.lang.AFn"})
+
+(defn stacktrace-entry->string
+  [{:keys [var name file line]}]
+  (let [name (or var name)]
+    (str name " (" file ":" line ")")))
+
+(defn stacktrace->message
+  "Takes a clojurefied stacktrace and returns a string representation of it for printing."
+  [stacktrace]
+  (->> stacktrace
+       (filter (fn [{:keys [flags class]}]
+                 (and (not (some #{"dup"} flags))
+                      (not (contains? stacktrace-classes-to-ignore class)))))
+       (map stacktrace-entry->string)
+       (str/join "\n")))
```

Comment:
This logic was basically copied from the TS code so the stack traces look the same.